.github/workflows/master.yml

```diff
@@ -118,6 +118,11 @@ jobs:
             pandas-version: 1.1.3
             pyarrow-version: 1.0.1
             default-index-type: 'distributed-sequence'
+          - python-version: 3.9
```

Comment:
I think we can just change the Python version in the existing matrix above instead of adding a new matrix.

Reply:
Could you provide an example of this?

Reply:
For example,

```
          - python-version: 3.6
            spark-version: 2.4.7
            pandas-version: 0.24.2
            pyarrow-version: 0.14.1
            logger: databricks.koalas.usage_logging.usage_logger
          - python-version: 3.6
            spark-version: 2.4.7
            pandas-version: 0.25.3
            pyarrow-version: 0.15.1
            default-index-type: 'distributed-sequence'
          - python-version: 3.7
            spark-version: 2.4.7
            pandas-version: 0.25.3
            pyarrow-version: 0.14.1
          - python-version: 3.7
            spark-version: 2.4.7
            pandas-version: 1.0.5
            pyarrow-version: 0.15.1
            default-index-type: 'distributed-sequence'
          - python-version: 3.8
            spark-version: 3.0.1
            pandas-version: 0.25.3
            pyarrow-version: 0.15.1
          - python-version: 3.9
            spark-version: 3.0.1
            pandas-version: 1.1.3
            pyarrow-version: 1.0.1
            default-index-type: 'distributed-sequence'
```